lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/kevinmason/Patron.java

```diff
@@ -0,0 +1,30 @@
+package com.codedifferently.lesson9.kevinmason;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Patron {
+  private String name;
+  private List<Book> checkedOutBooks;
```

Comment:
The `Library` should own the knowledge of whether a book is checked out or not, not the `Patron`.